action_plugins/play_sound/PlaySoundAction.qml

```diff
@@ -69,16 +69,15 @@ Item {
             text: "Volume"
         }
 
-        TextField {
+        SpinBox {
             id: _soundVolume
-
-            Layout.fillWidth: false
-            Layout.preferredWidth: 50
-            text: action.sound_volume
-            selectByMouse: true
-
-            onTextChanged: {
-                action.sound_volume = text
+            Layout.preferredWidth: 100
+            value: 50
```

Comment:
This won't show the correct value that the action has stored, you need to use the property binding stuff from QML to have the UI reflect the model's state. Though in your case that's really easy

`value: action.sound_volume`

Reply:
Yup, thanks - this was a left over from the first implementation.